dm/master/server.go

```diff
@@ -1985,3 +2263,341 @@ func (s *Server) workerArgsExtractor(args ...interface{}) (workerrpc.Client, str
 
 	return cli, worker, nil
 }
+
+// WatchRequest watches requests in etcd, and handle these request, and write response to etcd.
+func (s *Server) WatchRequest(ctx context.Context) {
+	watchCh := s.etcdClient.Watch(context.Background(), defaultOperatePath, -1)
+
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case wresp := <-watchCh:
+			if wresp.Err() != nil {
+				log.L().Warn("watch etcd failed", zap.Error(wresp.Err()))
+				continue
+			}
```

Comment:
should we need to handle the close of the channel?

Reply:
done in 9899502